src/ui/connectiondialog.cc

```diff
@@ -188,6 +189,14 @@ void ConnectionDialog::loadDefaultValues() {
       util::settings::connection::shutDownServerOnQuitDefault());
 }
 
+void ConnectionDialog::loadProfiles() {
```

Comment:
nit. if we have deleted our last profile `local server` will be entered in QComboBox, but it won't be selectable element of this QComboBox.